src/components/IOUConfirmationList.js

```diff
@@ -16,6 +16,7 @@ import SettlementButton from './SettlementButton';
 import ROUTES from '../ROUTES';
 import networkPropTypes from './networkPropTypes';
 import {withNetwork} from './OnyxProvider';
+import personalDetailsPropType from '../pages/personalDetailsPropType';
```

Comment:
NAB - should be plural since there's more than one prop type in this file - though this isn't your fault since you didn't create the original file :D

```suggestion
import personalDetailsPropTypes from '../pages/personalDetailsPropTypes';
```

Reply:
Good call. Updating.